README.md

```diff
@@ -217,6 +252,46 @@ Each cluster that will be connected must have Submariner installed within it. Yo
    |\<SERVICE_CIDR>|Service CIDR for Cluster|""|`10.43.0.0/16`|
    |\<NAT_ENABLED>|If in a cloud provider that uses 1:1 NAT between instances (for example, AWS VPC), you should set this to `true` so that Submariner is aware of the 1:1 NAT condition.|"false"|`false`|
 
+
+## Installation using operator
+
+An operator is a tool to package, manage and deploy any Kubernetes application. A submariner operator will soon be available on [OperatorHub.io](https://operatorhub.io).
+
+Current implementation builds submariner operator and deploys submariner in a [Kind setup](https://github.com/submariner-io/submariner/blob/master/scripts/kind-e2e/README.md).
+
+### Generating the Operator
+
+The current (developer-oriented) implementation dynamically generates the
+operator. This allows us to consume updates to the underlying best practices of
+the Operator SDK. It also results in a clear, working example of how to use the
+Operator SDK to create additional operators (perhaps for future parts of
+Submariner).
+
+```
+  cd operators/go
+  make codegen-operator
+```
+
+That will run the operator sourcecode generation logic in ./gen_subm_operator.sh
+
```

Comment:
I'd remove this paragraph, we are planning to get rid of operator generation. There's no real value and you don't need to generate it now, it's checked in the code already. I'd omit the whole section

Reply:
You can leave the build-operator part. Althought that will change soon too as we are moving that to a separate git repo.